tests/test_cli.py

```diff
@@ -291,77 +120,15 @@ def test_cli_arguments(self):
         self.maxDiff = None
         print(json.dumps(json_result, indent=2))
 
-        #Remove partial exception counts, because we can't guarantee that they'll be the same every time.
-        del json_result["results"][0]["summary_obj"]['partial_exception_counts']
+        #Remove partial unexpected counts, because we can't guarantee that they'll be the same every time.
+        del json_result["results"][0]["result_obj"]['partial_unexpected_counts']
```

Comment:
@mgasner 's patch to `partial_exception_counts` fixes this. (If the list is sorted, you can reliably test for the dictionary order of `partial_unexpected_counts`.)

We'll need to make sure we don't lose it after merging everything back in.

Reply:
I didn't think I'd added that in the first place, but I just removed those lines (so we *are* testing the partial_unexpected_counts).

Just a note here: there are cases (e.g. nonhashable values like a dict), where the partial_unexpected_counts cannot be computed.

Reply:
What should we do in that case...?